### ISSUE TITLE
npm WARN engine browserify-commit-sha@0.1.0: wanted: {"node":"0.12.x"} (current: {"node":"5.3.0","npm":"3.3.12"})

### ISSUE THREAD

**Issue by deathcap** [kumavis/browserify-commit-sha#5]
Installing browserify-commit-sha on nodejs 5.3.0 warns:

```
npm WARN engine browserify-commit-sha@0.1.0: wanted: {"node":"0.12.x"} (current: {"node":"5.3.0","npm":"3.3.12"})
```

due to browserify-commit-sha package.json:

```
  "engines": {
    "node": "0.12.x"
  },
```

otherwise, seems to work fine. Not sure off-hand of the proper fix (remove the engines clause entirely? add 5.3.0 or various other tested known-working engine versions?) but it is probably trivial :)

**Comment by kumavis** [comment]
yeah I guess I'll just remove the engines clause. but I'm not actively using this module anywhere atm.

**Comment by kumavis** [comment]
published as `0.1.1`
